Assets/Scripts/UI/Screens/OptionsScreen.cs

```diff
@@ -45,13 +54,14 @@ private void CreateElements()
             CreateAudioElements();
```

Comment:
I had it enabled but in the merge where disbled again. Let me commit a rearrangement of the icons because I think it will be overwritten language selection over volume selecion